clj/test/cljd/test_clojure/test_test.cljd

```diff
@@ -21,14 +21,19 @@
 (use-fixtures :once onetime-fixture)
 (use-fixtures :each repeated-fixture)
 
-(deftest mutator-test
+;; test order is not guaranteed, so both tests test both for mutation
+;; and the reset of the state by the :each fixture
+
+(deftest mutator-test-one
   (testing "Initial state"
-    (is (= {:base-data :foo} @test-state)
-        ":once fixtures are run before any tests and in the very end"))
+    (is (or (= {:base-data :foo} @test-state)
+            (= {:base-data :bar-2} @test-state))
+        ":once fixtures are run before any tests and in the very end.
+         if mutator-test-two ran first the value is expected to be :bar-2"))
```

Comment:
I don't understand: shouldn't the each fixtures have reset the state so that `:bar-2` is never visible?